### ISSUE TITLE
[firebase/auth]  passwordPolicyConfig doesn't exist on projectConfigManager

### ISSUE THREAD

**Issue by juozasm** [firebase/firebase-admin-node#2056]
Based on this google article I should be able to access password policy config:
https://cloud.google.com/identity-platform/docs/password-policy

I founded no trace of this config, in neither version of sdk from 9-11 and no beta pr preview release founded. 
Is this planned feature? Or its removed? How can I change password policies via sdk?

Here a code snipper I try using

```
export const getProjectConfig = () => {
    getAuth(firebaseApp).projectConfigManager().getProjectConfig()
        .then((response) => {
            console.log('Project password policy config: ', (response as any).passwordPolicyConfig);
        })
        .catch((error) => {
            console.log('Error getting project config:', error);
        });
};

const updateConfigRequest = {
    passwordPolicyConfig: {
        passwordPolicyEnforcementState: 'ENFORCE',
        forceUpgradeOnSignin: false,
        passwordPolicyVersions: [
            {
                customStrengthOptions: passwordCustomStrengthOptions,
            },
        ],
    },
};
export const updateProjectConfigWithPasswordPolicy = () => {
    getAuth().projectConfigManager().updateProjectConfig(updateConfigRequest as any).then((response) => {
        console.log('Updated password policy config for project: ', (response as any).passwordPolicyConfig);
    })
        .catch((error) => {
            console.log('Error updating project config:', error);
        });
};
```

**Comment by lahirumaramba** [comment]
Hi @juozasm, Thank you for filing this issue. You discovered a feature that is currently in Pre-GA! 😃 

The password policy API is currently in preview and does not have official support in the Admin SDK, yet. If you would like to try out the feature you might be able to cast the `passwordPolicyConfig` payload to `any` as shown in the code sample. Please keep in mind that pre-GA features have limited support and could change when ready for official release.

We will update the documentation once the feature is officially released. In the meantime, if you get a chance to try it out, we would love to hear your feedback.

**Comment by jingconan** [comment]
@lahirumaramba , just want to follow up on this feature.  I wonder if the passwordPolicyConfig is now officially supported?

I saw some documentation already on it. 

We found documentation in https://cloud.google.com/identity-platform/docs/password-policy and want to use this configuration to ensure strong password in our system. 

Our engineer did research but didn't find the configuration in the SDK yet. If it is not available yet, when do we expect to be avaliable? 

Looking forward to hearing from you. Thanks!
